src/Components/Activities/Activities.js

```diff
@@ -1,13 +1,32 @@
-import React from "react";
+import React, { useState } from "react";
+import './Activities.css';
+
+const Activities = ({ activities, updateActivity }) => {
+  const [currentActivityIndex, setCurrentActivityIndex] = useState(0);
+
+  const nextActivity = () => {
+    let nextIndex = currentActivityIndex + 1;
+    setCurrentActivityIndex(nextIndex);
+  }
+
+  const prevActivity = () => {
+    let prevIndex = currentActivityIndex - 1;
+    setCurrentActivityIndex(prevIndex);
+  }
+
+  const selectActivity = () => {
+    updateActivity(activities[currentActivityIndex].activity);
+  }
 
-const Activities = () => {
   return (
-    <>
-    <div>
-      <h2>Go find a spider!</h2>
-    </div>
-    <button>Go</button>
-    </>
+    <section className="activity-page">
+      <div className="activity-card">
+        {currentActivityIndex > 0 && <button onClick={() => prevActivity()}>←</button>}
+        <h2>{activities[currentActivityIndex].activity}</h2>
+        {currentActivityIndex < activities.length - 1 && <button onClick={() => nextActivity()}>→</button>}
+      </div>
+      <button id="go-button" onClick={() => selectActivity()}>Let's do it!</button>
```

Comment:
Can simplify the 3 activity methods here to just be the method names sans anon functions!